dir-walker/Containerfile

```diff
@@ -0,0 +1,27 @@
+# Building the example app from source
+# This is a contrived example that traverses through all directories in a given path and returns the largest 10 files. 
+FROM golang:1.19 AS build-stage
+WORKDIR /app
+ADD dir-walker.go Makefile .
+RUN make build
```

Comment:
Ah, I see. Since I'm building this in a prior stage(need the go toolchain), are you suggesting we let it install to /var/opt in that container? I guess I'd also have to change the line where I copy from the build stage to the final stage, if that makes sense

```#Drop in the app from the build stage. ADD can be used if the package is being directly added from the host and not being built. 
COPY --from=build-stage /var/opt/dir-walker /tmp```

Reply:
Yeah, through Makefile I would let it install in /var/opt/ and correct things in Containerfile later on just like we would deal with an rpm package that installs into /var/opt/ . Think of this example as something that is coming from somewhere else and already have logic of where things should be installed.